_includes/find-help/main-title.html

```diff
@@ -17,25 +17,31 @@
           <span class="icon-bar bottom-bar color-black"></span>
         </button>
       </div>
+      <div class="float-right">
```

Comment:
This is just my humble opinion but floats should be avoided at all possible costs. Floats knock the element out of the normal flow and hurts a11y and can bubble up js problems with events.

Flexbox solves most issues and the longer IE10 is in the rearview the more it should be the standard.